### DISCUSSION TITLE
Option for permanent FIM without manual shortcut?

### DISCUSSION THREAD

**Discussion by 3x3cut0r** [rjmacarthy/twinny#234] [Q&A]
Hey,
first of all great work!

My Question is:
Is there an option to acitivate permanent FIM suggestions like in other AI assistants?
I find it annoying to press the shortcut all the time.

**Comment by rjmacarthy** [comment]
It should be enabled by default? If not enable it in the config settings.

**Reply by 3x3cut0r** [reply]
I use default settings and I think it is enabled but then it just didn't work for me. I have to press the shortcut every time I want to have a FIM suggestion.
